bas_geoplot/interactive.py

```diff
@@ -353,86 +353,125 @@ def Paths(self,geojson,name,show=True,predefined=None,arrows=False,**kwargs):
         # Determining max travel-times of all paths
         for path in paths:
             points   = np.array(path['geometry']['coordinates'])
+            # Get index of any consecutive points that span over 180 deg in longitude
+            # i.e. get index of places which cross the antimeridian
+            split_at_idxs = [i + 1 for i in range(len(points)-1)
+                             if (np.abs(points[i][0] - points[i+1][0]) >= 180) ]
+            
+            segments = np.split(points, split_at_idxs)
+            extended_segments = []
+            for i, segment in enumerate(segments):
+
+                # If not the first segment, add the last point of the previous segment
+                # to the beginning of this segment to better visualise the path 
+                if i != 0:
+                    # Let this singular point be on the opposite side of the antimeridian
+                    orig_coords = segments[i-1][-1]
+                    if orig_coords[0] > 0: 
+                        rollover_coords = [orig_coords[0] - 360, orig_coords[1]]
+                    else:
+                        rollover_coords = [orig_coords[0] + 360, orig_coords[1]]
+                    
+                    segment = np.insert(segment, 0, rollover_coords, axis=0)
+                # If not the last segment, add the first point of the next segment
+                # to the end of this segment to better visualise the path 
+                if i != len(segments)-1:
+                    # Let this singular point be on the opposite side of the antimeridian
+                    orig_coords = segments[i+1][0]
+                    if orig_coords[0] > 0: 
+                        rollover_coords = [orig_coords[0] - 360, orig_coords[1]]
+                    else:
+                        rollover_coords = [orig_coords[0] + 360, orig_coords[1]]
+                    
+                    segment = np.insert(segment, -1, rollover_coords, axis=0)
+                # Add to list of 'extended' segments
+                extended_segments += [segment]
 
             start_wpt = path['properties']['from']
             end_wpt   = path['properties']['to']
-
             if p['data_name']:
                 try:
                     data_val = np.array(path['properties'][p['data_name']])*p['scaling_factor']
                 except:
                     continue
             else:
                 data_val = np.array(len(points))
+            
+            # For each series of points in each segment (defined by crossing antimeridian)
+            for points in extended_segments:
 
-            # Find the max value for this path for display in pop-up
-            path_max = np.max(data_val)
+                # Find the max value for this path for display in pop-up
+                path_max = np.max(data_val)
```

Comment:
Why is this now done inside the loop over the extended segments? I would've thought we only want to find the max value once per path?